hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/InvocationMonitor.java

```diff
@@ -212,6 +214,50 @@ long getLastMemberHeartbeatMillis(Address memberAddress) {
         return heartbeat == null ? 0 : heartbeat.get();
     }
 
+    private abstract class MonitorTask implements Runnable {
+        @Override
+        public void run() {
+            try {
+                run0();
+            } catch (Throwable t) {
+                inspectOutputMemoryError(t);
```

Comment:
Maybe this is a good opportunity to fix the name of this method? `inspectOutOfMemoryError`.